Scripts/ModelViewport.js

```diff
@@ -281,8 +269,39 @@ export default function ModelViewport(spec) {
   canvas.width = width;
   canvas.height = height;
   setupWebGL2();
-  initShaders();
-  loadModels();
+
+  const shdrDir = "Shaders/"; //FIXME: duplicated code from BRDFViewport
```

Comment:
Little thing, this should be ./Shaders/, so that if the project is hosted in a subdirectory, these will be located relative to index.html.